docs/Maths.md

```diff
@@ -226,6 +226,107 @@ Result:
+import cairosvg
+
+# Create a Pygal bar chart
+bar_chart = pygal.Bar()
+bar_chart.title = 'Browser usage evolution (in %)'
+bar_chart.x_labels = map(str, range(2002, 2013))
+bar_chart.add('Firefox', [None, None, 0, 16.6,   25,   31, 36.4, 45.5, 46.3, 42.8, 37.1])
+bar_chart.add('Chrome',  [None, None, None, None, None, None,    0,  3.9, 10.8, 23.8, 35.3])
+bar_chart.add('IE',      [85.8, 84.6, 84.7, 74.5,   66, 58.6, 54.7, 44.8, 36.2, 26.6, 20.1])
+bar_chart.add('Others',  [14.2, 15.4, 15.3,  8.9,    9, 10.4,  8.9,  5.8,  6.7,  6.8,  7.5])
+
+# Use CairoSVG to convert PNG from SVG of barchart
+svg_img_byte = BytesIO()
+cairosvg.svg2png(bar_chart.render(), write_to=svg_img_byte)
+
+# Set the position and size of the image in the PDF
+x = 50
+y = 50
+w = 100
+h = 70
+
+# Make the PDF
+pdf = FPDF()
+pdf.add_page()
+pdf.image(svg_img_byte, x=x, y=y, w=w, h=h)
+pdf.output('bar_chart.pdf')
+```
+The above code generates a pdf with title `bar_chart.pdf` file with following graph -
+![](pygal_chart_cairo.png)
+
+**!! Troubleshoot: !!**
+
+You may encounter `GTK` (Gnome Toolkit) errors while executing the above example in windows. Error could be like following -
+```
+OSError: no library called "cairo-2" was found
+no library called "cairo" was found
+no library called "libcairo-2" was found
+cannot load library 'libcairo.so.2': error 0x7e
+cannot load library 'libcairo.2.dylib': error 0x7e
+cannot load library 'libcairo-2.dll': error 0x7e
+```
+In this case install install `GTK` from [GTK-for-Windows-Runtime-Environment-Installer](https://github.com/tschoonj/GTK-for-Windows-Runtime-Environment-Installer/releases). Restart your editor. And you are all done.
+
+### Using svglib and reportlab (*A slower and purely pythonic implementation*) ### 
+```python
+import pygal
+from reportlab.graphics import renderPM
+from svglib.svglib import svg2rlg
+from fpdf import FPDF
+
+# Create a Pygal bar chart
+bar_chart = pygal.Bar()
+bar_chart.title = 'Sales by Year'
+bar_chart.x_labels = ['2016', '2017', '2018', '2019', '2020']
+bar_chart.add('Product A', [500, 750, 1000, 1250, 1500])
+bar_chart.add('Product B', [750, 1000, 1250, 1500, 1750])
+
+# Render the chart to an SVG and PNG file
+bar_chart.render_to_file('bar_chart.svg')
+drawing = svg2rlg("bar_chart.svg")
+renderPM.drawToFile(drawing, "bar_chart.png")
+
+# Set the position and size of the image in the PDF
+x = 50
+y = 50
+w = 100
+h = 70
+
+pdf = FPDF()
+pdf.add_page()
+pdf.image('bar_chart.png', x=x, y=y, w=w, h=h)
+pdf.output('bar_chart.pdf')
+
+# os.remove('bar_chart.png')  # It removes generated PNG file. Uncomment if needed
+# os.remove('bar_chart.svg')  # It removes generate SVG file. Uncomment if needed
+```
+The above example takes around 40 seconds to render the image. User who are using `reportlab` and `svglib` to work with `svg` images and are intended to render `svg` images `reportlab` and `svglib` can use this library. However, it does not garranty for a lesser time to accomplish the process.
```

Comment:
Those "40 seconds" depends on the user computer hardware & system: is it really meaningful to share it?

Reply:
You are right. I will remove this line.